webserver/src/test/java/org/jboss/test/arquillian/ce/webserver/WebserverTestBase.java

```diff
@@ -38,7 +38,7 @@
 import javax.websocket.Session;
 import javax.websocket.WebSocketContainer;
 
-import junit.framework.Assert;
+import org.junit.Assert;
```

Comment:
It's a convention that imports are alphabetically ordered. So, this line would need to jump to after the `import org.jboss.shrinkwrap.api.spec.WebArchive;` line. Eclipse, for instance, does that automatically for you: `CTRL+SHIFT+o`.